utils/zerovec/derive/src/lib.rs

```diff
@@ -59,6 +98,10 @@ pub fn make_ule(attr: TokenStream, item: TokenStream) -> TokenStream {
 ///
 /// The type must be `PartialEq` and `Eq`.
 ///
+/// `EncodeAsVarULE` and `ZeroFrom` are useful for avoiding the need to deal with the VarULE type directly. In particular,
```

Comment:
Nit: links and code formatting, also above. I think the Wikipedia guideline is to link the first occurrence in each paragraph, I think we should do that.

Reply:
We can't in this crate, the dependencies are backwards.

Reply:
One way to do it would be to document the reexports with links instead.

Reply:
I went ahead and did this. People have almost no reason to use the derive crate directly.